Utilities/rANS/include/rANS/internal/ReverseSymbolLookupTable.h

```diff
@@ -38,11 +38,6 @@ class ReverseSymbolLookupTable
   {
     LOG(trace) << "start building reverse symbol lookup table";
 
-    if (stats.size() == 1) {
-      LOG(warning) << "SymbolStatistics of empty message passed to " << __func__;
-      return;
-    }
-
     mLut.resize(bitsToRange(probabilityBits));
```

Comment:
@MichaelLettrich please check here. For empty dictionary (stats.size()==1) we still create LUT of `1<<probabilityBits` size filled by 0s, and this LUT is queried here: https://github.com/shahor02/AliceO2/blob/139aec32c41a94e386c6f21c3dd87e5c85f877d3/Utilities/rANS/include/rANS/LiteralDecoder.h#L70-L73 to check the `isRareSymbol`. So the `streamSymbol` will be always 0 and on top of that with empty dict. any symbol is rare.
So I don't fully understand the logic here.